src/lib/PostgresMetaFunctions.ts

```diff
@@ -286,7 +276,7 @@ export default class PostgresMetaFunctions {
               ) AS arr
             ) AS split_args
           ) args
-    ) ${terminateCommand ? ';' : ''}`
+    )`
```

Comment:
@soedirgo intentionally left off the semicolon due to it being set `INTO` function in update function's plpgsql statement.